adam-core/src/main/java/org/bdgenomics/adam/io/InterleavedFastqInputFormat.java

```diff
@@ -127,13 +123,15 @@ public InterleavedFastqRecordReader(Configuration conf, FileSplit split) throws
          */
         private void positionAtFirstRecord(FSDataInputStream stream) throws IOException {
             Text buffer = new Text();
-            int bytesRead = 0;
-            
+
+            if (true) { // (start > 0) // use start>0 to assume that files start with valid data
```

Comment:
Correct. I'm hoping to do more of a cleanup later (e.g., similar to the cleanup that I did in #611), but wanted to get this back to working first.